src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -240,11 +247,11 @@ public synchronized static void parse(DLNAMediaInfo media, InputFile inputFile,
 						}
 
 						value = MI.Get(audio, i, "BitDepth");
-						if (!value.isEmpty()) {
-							try {
+						if (isNotBlank(value) || !value.contains("Bit")) {
+							if (!value.contains("/")) {
 								currentAudioTrack.setBitsperSample(Integer.parseInt(value));
-							} catch (NumberFormatException nfe) {
```

Comment:
@Sami32 It's bad to remove the catch here, ```Integer.parseInt()``` will throw if ```value``` is anything but a valid integer. That will terminate the thread and disconnect the renderer.